src/extensions/screencopy.rs

```diff
@@ -0,0 +1,30 @@
+//! Support for the wlroots Screencopy (Version 1) Protocol
+
+use wayland_sys::server::wl_display as wl_server_display;
+use wlroots_sys::{wl_display, wlr_screencopy_manager_v1, wlr_screencopy_manager_v1_create,
+    wlr_screencopy_manager_v1_destroy};
+
+#[derive(Debug)]
+/// Manager that offers requests to start capturing from a source
+pub struct Manager {
+    manager: *mut wlr_screencopy_manager_v1
```

Comment:
This protocol is unstable (due to the `z` in its name and the fact it has a "v1" in it). It's possible there are going to be many versions of this that will need to be supported as the final decision is made on the stabilization. This happened with xdg-shell (v5 and v6 existed for a while, and now _most_ use the stable version).

So far the way I've encoded this is by adding a `V#` in the type somewhere, like I did with xdg shell. So we should name this `ManagerV1` or even better `ZManagerV1`. 

You should also point out in the docs that this is unstable and you should link to the protocol (I've been flubbing on the documentation thus far, so I won't hold against you if you do the same).

We cannot use a feature flag, as different versions of the same protocol can exist at the same time.

Reply:
Gotcha. I'll get this fixed up later today. Thanks for the feedback. I'll keep it in mind with the other unstable protocols.